tools/crm_shadow.c

```diff
@@ -252,18 +290,6 @@ shadow_teardown(char *name)
     free(our_prompt);
 }
 
-static bool
```

Comment:
Despite the commit message saying these all fit nicely together, I might split some of these changes up on the next update. It might have gotten more complex since I wrote that.